_sections/act_and_prepare/en/02-do/monitorsymptoms.md

```diff
@@ -0,0 +1,36 @@
+Compiled from [this WHO report](https://www.who.int/docs/default-source/coronaviruse/who-china-joint-mission-on-covid-19-final-report.pdf)   
+Common symtoms
+| symptom | percentage | symptom | percentage | symptom | percentage |
+| --- | --- | --- | --- | --- | --- |
+| Fever | 88% |Fatigue|38%|Shortness of breath|18%|
+| Dry cough |68% |Phlegm production|33.4%|Sore throat |14%|
+|Headaches |14%|Muscle aches|14%|Chills |11%|
+
+Gastrointestinal Symptoms (rare): 
+Diarrhea, nausea, vomiting, or abdominal discomfort are all possible early symptoms. They may appear a couple of days BEFORE the respiratory symptoms. 
+
+Timeline  
+1) Mild cases usually clear up in 7 days (no hospitalization). 
+2) It can take 2-14 days for the first covid19 symptom to develop. 88% of them eventually have fever. 
+For nearly half of the hospitalized patients, fever may not develop before you need to be admitted to hospital. 
+3) By day 5, patients of preconditions may have trouble breathing. 
+
+Recovery time: (for all patients that develop pneumonia) \
+Mild: a few days.     
+Severe: 2-1/2 weeks.  
+Critical (with ARDS): 30-40% fatal. Recovery could take months. 
+
+Using tracking symptom apps:  \
+US: covidaware.me\
+UK: https://covid.joinzoe.com. \
+CA: flatten.ca \
+Japan: www.coronatracker.com   
+
+
+
+
+
+
+
+
+
```

Comment:
We don't need quite so many empty newlines here. One should suffice.

Reply:
just removed all empty lines

Reply:
LGTM